### ISSUE TITLE
Tablet support

### ISSUE THREAD

**Issue by brianmay** [EddieRingle/hubroid#74]
Doesn't appear to be available in the Market for my Galaxy 10.1; Not sure if this is due to HoneyComb or the larger screen size. 

Also suspect screen design could use optimizing for tablet's larger screen.

**Comment by EddieRingle** [comment]
Yes, I have it filtered out for Honeycomb and up as well as for larger screen sizes. Previously I simply didn't have the resources to test Hubroid on those devices, so I had to make choices.

School has been getting in the way a lot, but I got a tablet for Christmas and intend on finally getting Hubroid up and running on larger screens. I think this'll coincide with ICS support, perhaps making use of ActionBarSherlock 4.0. :)
